openapi/v2/core-data.yaml

```diff
@@ -1346,6 +1346,18 @@ paths:
               examples:
                 400Example:
                   $ref: '#/components/examples/400Example'
+        '416':
+          description: "Request range is not satisfiable"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                400Example:
```

Comment:
why not calling it 416Example?

Reply:
It's a typo since 416Example was added.  Fixed all occurrences of this typo.